contracts/examples/Earth.sol

```diff
@@ -108,10 +108,10 @@ contract Earth is IbcUniversalPacketReceiverBase {
         external
         onlyUCH
         onlyAuthorizedChannel(channelId)
-        returns (AckPacket memory ackPacket)
+        returns (AckPacket memory ackPacket, bool skipAck)
```

Comment:
we should update this method's documentation with the extra @return type it now returns

Reply:
@mvpoyatt also in the ibc receiver interface which all the dapps implement, if you haven't done so already